share/lib/python/neuron/gui.py

```diff
@@ -26,15 +24,6 @@ def start():
     _lock.release()
 
 
-"""
-This allows one to temporarily disable the process_events loop using a construct
-like:
-from neuron import gui
-with gui.disabled():
-    something_that_would_interact_badly_with_process_events()
-"""
```

Comment:
I don't know if this is still current, but if so, it should either stay here, or be moved to the docstring of `disabled` below. If it's not current, just ignore this comment.

Reply:
I added back all the comments.